diff -ur libkolab-0.4.2/cmake/modules/SWIGUtils.cmake libkolab-0.4.2.swig-cmake/cmake/modules/SWIGUtils.cmake
--- libkolab-0.4.2/cmake/modules/SWIGUtils.cmake	2013-04-11 10:40:10.000000000 +0100
+++ libkolab-0.4.2.swig-cmake/cmake/modules/SWIGUtils.cmake	2013-06-17 20:39:57.336180457 +0100
@@ -16,7 +16,7 @@
 
     #Compile PHP Bindings
     # Since there is no php library we can't compile with -Wl,--no-undefined
-    set(CMAKE_SHARED_LINKER_FLAGS "")
+    set(CMAKE_SHARED_LINKER_FLAGS "-Wl,-z,relro,--as-needed")
 
     if (APPLE)
         set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -flat_namespace -undefined suppress" )
