From 0c072d69d677fa7cdad65af563994f6a0098003f Mon Sep 17 00:00:00 2001
From: Thomas Bruederli <bruederli@kolabsys.com>
Date: Thu, 21 May 2015 17:42:43 +0200
Subject: [PATCH 2/7] Make locale encoding the default sys encoding; resolves
 T249

---
 pykolab/cli/__init__.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pykolab/cli/__init__.py b/pykolab/cli/__init__.py
index 56f121b..4843aba 100644
--- a/pykolab/cli/__init__.py
+++ b/pykolab/cli/__init__.py
@@ -63,6 +63,10 @@ class Cli(object):
         for cmd_component in to_execute:
             sys.argv.pop(sys.argv.index(cmd_component.replace('_','-')))
 
+        # force default encoding to match the locale encoding (T249)
+        reload(sys)
+        sys.setdefaultencoding(locale.getpreferredencoding() or 'utf-8')
+
         # wrap sys.stdout in a locale-aware StreamWriter (#3983)
         sys.stdout = codecs.getwriter(locale.getpreferredencoding())(sys.stdout)
 
-- 
1.9.3

