--- a/debian/libcalendaring-dev.install	2015-03-04 09:22:05.693065915 +0100
+++ b/debian/libcalendaring-dev.install	2015-03-04 09:22:17.592127918 +0100
@@ -1,2 +1,3 @@
 usr/include/*
 usr/lib/lib*.so
+usr/lib/lib*.a
--- a/debian/rules	2015-03-04 12:24:02.005134594 +0100
+++ b/debian/rules	2015-03-04 12:23:28.647765863 +0100
@@ -11,3 +11,9 @@
 
 %:
 	dh $@ 
+
+# we need the kimaptest.a in /usr/lib, but it will be removed by dh_strip
+override_dh_strip:
+	pwd
+
+.PHONY: override_dh_strip
